hack/deploy-cluster.sh

```diff
@@ -83,10 +89,37 @@ aks-engine deploy --subscription-id ${SUBSCRIPTION_ID} \
   --client-secret ${CLIENT_SECRET}
 echo "Kubernetes cluster deployed. Please find the kubeconfig for it in _output/"
 
-export KUBECONFIG=_output/$(ls -t _output | head -n 1)/kubeconfig/kubeconfig.$LOCATION.json
-echo "Kubernetes cluster deployed. Please find the kubeconfig at $KUBECONFIG"
 
-# Deploy AzureDisk CSI Plugin
+check=$(aks-engine version)
```

Comment:
I think below changes are not needed, right? there is alreayd aks-engine deploy on above L82

Reply:
yes sir but previously, it was done